scripts/node-redirect.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
My confidence as a reviewer of the redirect scripts is pretty low.. can Yuval or someone with more expertise step in?